.github/workflows/canary-aarch64-linux.yml

```diff
@@ -12,7 +12,7 @@ on:
       djl-version:
         description: 'djl version to test'
         required: false
-        default: '0.19.0-SNAPSHOT'
```

Comment:
We should remove -SNAPSHOT in demo repo, we use -SNAPSHOT only when for new features

Reply:
OK